### DISCUSSION TITLE
rofi cache?

### DISCUSSION THREAD

**Discussion by se7en-x230** [prasanthrangan/hyprdots#918] [Q&A]
I have a strange rofi problem on a new install. Arch + Hyperdots

I have custom scripts in .local/bin/hypr which start well but I wanted to modify the script to show a input filed text 
   
    `rofi_cmd="(rofi -dmenu -theme-str 'entry { placeholder: \"Copy...\";}')"  `

1. question: is this the right way to add text 
2. question: as soon as I change some code and run it I get the old version and not the modified one, as if rofi had a cache

I guess the syntax/quotes are not correct

**Comment by RookiexCookie** [comment]
uh can you kill it and repress the hotkey
see if that does it

**Comment by kRHYME7** [comment]
@se7en-x230  

try running this 

``` echo -e "Use\nDouble\nQoute" | rofi -dmenu -theme-str  "entry { placeholder: \"Copy...\";}"  ```
